changelogs/client_server/newsfragments/2073.clarification

```diff
@@ -0,0 +1 @@
+Deduplicate the definition of `RoomKeysUpdateResponse`.
```

Comment:
this should maybe be `/internal/`? I don't think it has any effect on the rendered output or OpenAPI definitions?

Reply:
Indeed, done.